node-api/handlers/beacon/routes.go

```diff
@@ -23,15 +23,11 @@ package beacon
 import (
 	"net/http"
 
-	"github.com/berachain/beacon-kit/log"
 	"github.com/berachain/beacon-kit/node-api/handlers"
 )
 
 //nolint:funlen // routes are long
-func (h *Handler) RegisterRoutes(
-	logger log.Logger,
-) {
-	h.SetLogger(logger)
+func registerRoutes(h *Handler) {
```

Comment:
unexported and made a free function, since this does not need to be called anymore outside of handlers (in `Server`), but it is only called in handler constructors